meteor/.meteor/release

```diff
@@ -1,2 +1 @@
-METEOR@1.11
-# Note: Remember to update the base docker image in .circleci/config.yml when updating this!
```

Comment:
Could we keep this comment? I think it'll help future updaters.
(I know, it was probably removed when doing `meteor update`, but still)